web/modules/custom/farm_nfa/js/farmOS.map.behaviors.farm_nfa_plan_locations_layers.js

```diff
@@ -1,12 +1,73 @@
 (function () {
   farmOS.map.behaviors.farm_nfa_plan_locations_layers = {
-    attach: function (instance) {
+    attach: async function (instance) {
       var url = new URL('/nfa-assets/geojson/' + instance.farmMapSettings.plan, window.location.origin + drupalSettings.path.baseUrl)
       var newLayer = instance.addLayer('geojson', {
         title: Drupal.t('Locations'),
         url,
         color: 'orange',
       })
+
+      // Add a layer for fire alerts in the GFW plan tab
+      let gfwRegex = a = /\/plan\/\d+\/gfw/;
+      let windowUrl = window.location.href;
+      if(windowUrl.match(gfwRegex)) {
```

Comment:
Please make sure you're following the coding standards for javascript as well

Reply:
surely